docs/DeveloperGuide.md

```diff
@@ -737,6 +737,43 @@ Use case ends.
 
 <br/>
 
+#### UC11: Undo most recent undoable command
+
+**Guarantees** The most recent undoable command (add, delete, edit, sort, clear) requested will be undone.
+
+**MSS:**
+
+**1.** User requests to undo.
+
+**2.** SimplyKitchen displays the food list before the most recent undoable command was requested.
+
```

Comment:
Should there be a "Use case ends." at the end of MSS? Same for line 767.

Reply:
Thanks! Updated.